src/io_uring/src/lib.rs

```diff
@@ -27,8 +30,12 @@ pub type Result<T> = std::result::Result<T, Error>;
 
 #[derive(Debug)]
 pub enum Error {
+    Disabled,
     CQueue(CQueueError),
+    Enable(IOError),
     RegisterEventfd(IOError),
+    RegisterAfterEnable,
+    RegisterRestrictions(IOError),
     RegisterFile(IOError),
     Setup(IOError),
     SQueue(SQueueError),
```

Comment:
Can we have all these errors sorted alphabetically in order to maintain a better visibility?

Reply:
done